### ISSUE TITLE
Program is showing the chunks I spend the most time on as deleted

### ISSUE THREAD

**Issue by iitaliics** [Querz/mcaselector#10]
Hi, the program is showing the chunks I generally spend the most amount of time in as black, and there is no way to know what i would or wouldn't be deleting. 
![overworld](https://user-images.githubusercontent.com/45936707/56880027-7c0b4080-6a9e-11e9-89f7-ab7dc352cdee.PNG)
![nether](https://user-images.githubusercontent.com/45936707/56880028-7ca3d700-6a9e-11e9-887f-febd6ee42965.PNG)

**Comment by Querz** [comment]
If you enable "debug" mode in the settings, could you send me the console output either here as a Gist or on Pastebin? Alternatively you can also send me the world files `r.0.0.mca`, `r.-1.-1.mca`, `r.0.-1.mca` and `r.-1.0.mca` of either your overworld or the nether, assuming that the black area is around x=0, z=0 in your world.

**Comment by iitaliics** [comment]
I realize this tool was not meant for worlds on 1.14, which is what I was using it on, so that is probably the cause, thanks for your concen

**Comment by Querz** [comment]
The goal is to update the tool to also work with 1.14 worlds, and I could not reproduce this issue with a newly generated 1.14 world (at least not within a few minutes of generating the world).

**Comment by iitaliics** [comment]
In that case I'll send the files as I don't know where the console log is. While the black parts are around x=0 y=0, they are also everywhere new chunks are loaded as I deleted the cache for me to see.

Hold on I just deleted a bunch of useless chunks, played the game for a bit and then deleted the cache on mcaselecter and some of them are appearing. 
![okay](https://user-images.githubusercontent.com/45936707/56883269-82eb8080-6aa9-11e9-8e4f-4c868006a2f9.PNG)






[region.zip](https://github.com/Querz/mcaselector/files/3126440/region.zip)
